src/widgets/geo-search/__tests__/geo-search-test.js

```diff
@@ -30,17 +30,11 @@ describe('GeoSearch', () => {
     getZoom: jest.fn(),
     setZoom: jest.fn(),
     getBounds: jest.fn(() => ({
-      getNorthEast: jest.fn(),
-      getSouthWest: jest.fn(),
-    })),
-    getProjection: jest.fn(() => ({
-      fromPointToLatLng: jest.fn(() => ({
-        lat: jest.fn(),
-        lng: jest.fn(),
+      getNorthEast: jest.fn(() => ({
+        toJSON: jest.fn(() => ({})),
       })),
-      fromLatLngToPoint: jest.fn(() => ({
-        x: 0,
-        y: 0,
+      getSouthWest: jest.fn(() => ({
+        toJSON: jest.fn(() => ({})),
```

Comment:
valuable to have real possible values here or not?

Reply:
We do have valuable values here later on but not by default - mostly because we don't care about the value.

> https://github.com/algolia/instantsearch.js/pull/3158/files#diff-bc6fb9f0c9db3d39fdc54426badeeed3R1254